readme.md

```diff
@@ -34,26 +34,63 @@ A starter app is available on [codesandbox](https://codesandbox.io/s/github/matt
 ```shell
 npm install haunted
 ```
+
 For Internet Explorer 11, you'll need to use a proxy polyfill, in addition to the usual webcomponentsjs polyfills. 
+
 eg.
 ```html
-<script type="text/javascript" src="https://cdn.jsdelivr.net/npm/proxy-polyfill@0.3.0/proxy.min.js"></script> 
+<script src="https://cdn.jsdelivr.net/npm/proxy-polyfill@0.3.0/proxy.min.js"></script> 
 ```
-For a full example see - https://github.com/crisward/haunted-ie11
+
+For a full example with Internet Explorer 11, see - https://github.com/crisward/haunted-ie11
 
 You can also use Custom Elements without Shadow DOM if you wish.
 eg.
+
+```js
+component(() => html`...`, HTMLElement, { useShadowDOM: false }));
 ```
-component(() => html`...`, HTMLElement, {useShadowDOM: false}))
+
+### Importing
+
+__Haunted__ can be imported just like any other library when using a bundler of your choice:
+
+```js
+import { component, html, useState } from 'haunted';
 ```
 
-### Builds
+The main entry point is intended for [lit-html](https://github.com/Polymer/lit-html) users.
+
+#### lighterhtml, etc
+
+If you are using [lighterhtml](https://github.com/WebReflection/lighterhtml) or [hyperHTML](https://github.com/WebReflection/hyperHTML) then instead import `haunted/core`. This export gives you a function that creates Hooks that work with any template library.
+
+```js
+import haunted, { useState } from 'haunted/core';
```

Comment:
we should advocate to use `.js` extensions for importing, otherwise it will fail in the browser

omitting extensions is a non-standard that some bundlers have been doing for a long time, but it's not the best practice moving forward, and won't even execute in browsers today

```js
import haunted, { useState } from 'haunted/core.js';
```

Reply:
Bare module specifiers don't work in the browser either. So I don't think adding .js adds any value. Using bare specifiers assumes you're using some preprocessing. The haunted.js docs are intended for browser usage and therefore include the full url.

Reply:
that's fair enough... *for now!*

you can tell i can't wait for the sweet days when import maps become ubiquitous ;)